From 84e77a0a05733153b3dea8141feeae96bbb07f8b Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Tue, 2 Nov 2021 09:39:25 +0100
Subject: [PATCH 0284/1695] plat-stm32mp1: enable CFG_ARM_GIC_PM in conf.mk

Add the CFG_ARM_GIC_PM support for stm32mp1 platform.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a02631c582b255d2589b6c2ab177f4143307bc4d plat-stm32mp1: enable CFG_ARM_GIC_PM in conf.mk]
Signed-off-by: Etienne Carriere <etienne.carriere@st.com>
Change-Id: Ie22640f71967ae440d1cc53da4ec9569c552d196
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index def49aa28..0c20b351f 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -139,6 +139,7 @@ endif
 
 include core/arch/arm/cpu/cortex-a7.mk
 
+$(call force,CFG_ARM_GIC_PM,y)
 $(call force,CFG_DRIVERS_CLK,y)
 $(call force,CFG_DRIVERS_CLK_DT,y)
 $(call force,CFG_DRIVERS_GPIO,y)
-- 
2.47.2

